objects/client.c

```diff
@@ -3446,17 +3446,9 @@ client_class_setup(lua_State *L)
      */
     signal_add(&client_class.signals, "request::activate");
     /**
-     * @signal request::fullscreen
+     * @signal request::geometry
```

Comment:
Could you add some words about the argument(s) to the signal? The C code seems to emit this with a single string argument, but the handler function accepts extra hints.